modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm

```diff
@@ -186,6 +186,8 @@
 	var/returned_string = "You see a [lowertext(genital_name)] vagina."
 	if(lowertext(genital_name) == "cloaca")
```

Comment:
Then if we create the `desc_string` on the genital sprite_accessory datum, maaybe this coupled with suggested change above would handle it

```suggestion
	if(gas && gas.desc_string)
		returned_string = gas.desc_string
```